### PR TITLE
Improve contributor display

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: src/main/java/com/fabriccommunity/spookytime/util/Contributors.java [hunk 1]
```diff
@@ -1,62 +1,72 @@
...
 	private static void initContributors() {
-		/*
-		 * Here is an example of how to add a user to the list
-		 *
-		 * USERS_AND_FORMATTING.put("Github-Username/MC Username", Formatting.DARK_RED);
-		 *
-		 * Note that Magic, Underline and bold formattings are striped to make sure
-		 * someone doesn't mess up.
-		 */
+		ModMetadata metadata = FabricLoader.getInstance().getModContainer(SpookyTime.MOD_ID).orElseThrow(() -> new IllegalStateException("Spooky time mod broke!")).getMetadata();
```
Comment (Draylar) [inline src/main/java/com/fabriccommunity/spookytime/util/Contributors.java]: Could we change the string out for a translation? While I have a hard time seeing this error ever being thrown, a more appropriate message could be "Fabric failed to load Spooky Time's mod metadata! Please report this as a bug."

**Review by Draylar — CHANGES_REQUESTED** [review]

**Comment by Draylar** [comment]
Looking great!

File: src/main/java/com/fabriccommunity/spookytime/util/Contributors.java (hunk 1 above)
Comment (liach) [reply src/main/java/com/fabriccommunity/spookytime/util/Contributors.java]: Hmm, translation won't be available on the dedicated server 🤔 
And if this does happen it is a loader bug, not really one of ours

Comment (Draylar) [reply src/main/java/com/fabriccommunity/spookytime/util/Contributors.java]: Ah, didn't know translations couldn't be used in this context.

Comment (liach) [reply src/main/java/com/fabriccommunity/spookytime/util/Contributors.java]: Fixed. Now I'm asking people to shout at loader (if this broke, it is indeed the loader's fault)

**Review by Shnupbups — APPROVED** [review]

**Review by ImAK9 — APPROVED** [review]

**Pull request merged**